docs/_newsfragments/1194-cors.feature.rst

```diff
@@ -0,0 +1,3 @@
+A new kwarg, ``cors_enable``, was added to the :class:`falcon.API` initializer.
```

Comment:
Might want to change this to `falcon.App` depending on how #1592 lands.

Reply:
Let's handle this in #1592 IMHO (serializable workflow :slightly_smiling_face: )